### DISCUSSION TITLE
Add a Indian Horoscope generating tool to blog

### DISCUSSION THREAD

**Discussion by nishant31in** [community/community#68890] [Programming Help]
I want to add a Janam Kundli horoscope generation tool to my blog https://indianbabynames12.blogspot.com/

I didn't find a suitable code which can accept Date of Birth and Place of Birth and Birth Time and generate a kundli on clicking submit button. Most codes I found didn't work. Is it a server side feature that is needed to generate Kundli online or is is possible with simple CS and JS codes?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @nishant31in ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
